.github/workflows/unix_unit_tests.yml

```
@@ -13,7 +13,7 @@
       strategy:
         matrix:
           os: [ubuntu-latest,macos-latest]
-          python-version: [3.8, 3.9, '3.10']
+          python-version: [3.8, 3.9, '3.10', 3.11, 3.12]
```

```suggestion
          python-version: [3.8, 3.11, 3.12]
```

Really great work getting this working @kmilo9999!

As per our discussion, I think these will be the versions we stick with for now. Definitely will be keeping in mind the idea of having a separate branch for testing the full matrix in the future

---

Ah nice catch. I was discussing with a colleague that one possibility is to have a separate full-matrix workflow that tests the main branch as a cron job.  You can set up a workflow to run on a monthly 2-week basis or something.

---

Should I remove `3.9 `and `3.10` from the windows workflow too?

---

Yes

---

Done